tests/zfs-tests/tests/functional/delegate/setup.ksh

```diff
@@ -44,6 +44,11 @@ if ! is_linux; then
 	fi
 fi
 
+if is_freebsd; then
+	# To pass user mount tests
+	log_must sysctl vfs.usermount=1
+fi
```

Comment:
Should this be reset on clean to avoid impacting other tests?

Reply:
Done